keyboards/alectardy98/alpinenoon/keymaps/alec/keymap.c

```diff
@@ -0,0 +1,219 @@
+                SEND_STRING(SS_LCTRL("w"));                                                   //Mac
+                } else {}break;
+            case DZRO:
+            if (record->event.pressed) {
+                SEND_STRING("00");
+                } else {}break;
+            case ATAB:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LALT(SS_TAP(X_TAB)));
+                } else {}break;
+            case LPRC:
+            if (record->event.pressed) {
+                SEND_STRING("(");
+                } else {}break;
+            case RPRC:
+            if (record->event.pressed) {
+                SEND_STRING(")");
+                } else {}break;
+                //Discord Section
+                case THIS:
+                if (record->event.pressed) {
+                    SEND_STRING(":this:" SS_TAP(X_ENTER) );
+                    } else {}break;
+                case OOF:
+                if (record->event.pressed) {
+                    SEND_STRING(":oof:" SS_TAP(X_ENTER));
+                    } else {}break;
+                case PIKA:
+                if (record->event.pressed) {
+                    SEND_STRING(":surprised:" SS_TAP(X_ENTER));
+                    } else {}break;
+                case SCAT:
+                if (record->event.pressed) {
+                    SEND_STRING(":smiley_cat_spanish:" SS_TAP(X_ENTER));
+                    } else {}break;
+                case FCAT:
+                if (record->event.pressed) {
+                    SEND_STRING(":smiley_cat_fancy:" SS_TAP(X_ENTER));
+                    } else {}break;
+                case HART:
+                if (record->event.pressed) {
+                    SEND_STRING(":heart:" SS_TAP(X_ENTER));
+                    } else {}break;
+                case DROL:
+                if (record->event.pressed) {
+                    SEND_STRING(":drooling_face:" SS_TAP(X_ENTER));
+                    } else {}break;
+                case MONY:
+                if (record->event.pressed) {
+                    SEND_STRING(":smiley_takemoney:" SS_TAP(X_ENTER));
+                    } else {}break;
+                case FHAT:
+                if (record->event.pressed) {
+                    SEND_STRING(":smiley_tip_fedora:" SS_TAP(X_ENTER));
+                    } else {}break;
+                case SPIT:
+                if (record->event.pressed) {
+                    SEND_STRING(":smiley_spit_surprised:" SS_TAP(X_ENTER));
+                    } else {}break;
+                case KING:
+                if (record->event.pressed) {
+                    SEND_STRING(":pepe_king_royal:" SS_TAP(X_ENTER));
+                    } else {}break;
+        }
+    }
+#ifdef CONSOLE_ENABLE //Console Debug
+    uprintf("KL: kc: 0x%04X, col: %u, row: %u, pressed: %b, time: %u, interrupt: %b, count: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed, record->event.time, record->tap.interrupted, record->tap.count);
```

Comment:
Not sure that %b works anymore. 
```suggestion
    uprintf("KL: kc: 0x%04X, col: %u, row: %u, pressed: %1u, time: %u, interrupt: %1u, count: %u\n", keycode, record->event.key.col, record->event.key.row, record->event.pressed, record->event.time, record->tap.interrupted, record->tap.count);
```